content/pages/training/2024/08-nasatops.rst

```diff
@@ -0,0 +1,64 @@
+============================================================
```

Comment:
`workshop/` may be a more inclusive name for the directory, while I am ok with `training/`.